Bond/Bond+Arrays.swift

```diff
@@ -663,50 +663,50 @@ private class DynamicArrayDeliverOnProxy<T>: DynamicArray<T> {
     super.init([])
     
     bond.willInsertListener = { [unowned self] array, i in
-      dispatch_async(queue) { [weak self] in
```

Comment:
From the Swift 2 prerelease book:

> Unlike a weak reference, however, an unowned reference is assumed to _always_ have a value. Because of this, an unowned reference is aways defined as a non-optional type.

So what @sync has done is correct.